onebusaway-android/src/main/java/org/onebusaway/android/ui/HomeActivity.java

```diff
@@ -586,9 +584,10 @@ private void autoShowWhatsNew() {
      *
      * @param stop   the ObaStop that obtained focus, or null if no stop is in focus
      * @param routes a HashMap of all route display names that serve this stop - key is routeId
+     * @param location the user touch location on the map
      */
     @Override
-    public void onFocusChanged(ObaStop stop, HashMap<String, ObaRoute> routes) {
+    public void onFocusChanged(ObaStop stop, HashMap<String, ObaRoute> routes, Location location) {
         // Check to see if we're already focused on this same stop - if so, we shouldn't do anything
         if (mFocusedStopId != null && stop != null &&
                 mFocusedStopId.equalsIgnoreCase(stop.getId())) {
```

Comment:
We should clarify the below comment `// A particular stop lost focus (e.g., user tapped on the map), so hide the panel` to be `// No stop is in focus (e.g., user tapped on the map), so hide the panel`